receiver/vcenterreceiver/metadata.yaml

```diff
@@ -441,6 +441,13 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
     attributes: []
+  vcenter.vm.cpu.readiness:
+    enabled: true
```

Comment:
I believe this is going to have to start as false for a new addition.

This will also need a warning that it will be enabled by default in two releases (https://github.com/open-telemetry/opentelemetry-collector/blob/main/cmd/mdatagen/metadata-schema.yaml)

Reply:
Can resolve